crates/bevy_ecs/examples/world_diagnostic.rs

```diff
@@ -0,0 +1,115 @@
+//! In this example, we use a system to print diagnostic information about the world.
```

Comment:
It might be good to elaborate more on this. For example, what diagnostic info do we get and why is this useful to see?